src/clif-util.rs

```diff
@@ -321,9 +321,6 @@ fn main() {
                     rest_cmd.is_present("print-size"),
                     rest_cmd.is_present("time-passes"),
                     rest_cmd.is_present("value-ranges"),
-                    rest_cmd.is_present("enable-simd"),
-                    rest_cmd.is_present("enable-multi-value"),
-                    rest_cmd.is_present("enable-reference-types"),
```

Comment:
I think then that we should remove the `add_enable_simd_flag`, `add_enable_multi_value`, `add_enable_reference_types_flag` functions.

Reply:
Done!